clustering/marathon.py

```diff
@@ -0,0 +1,555 @@
+    default: "present"
+    description:
+      - The operation to perform.
+
+  username:
+    required: false
+    description:
+      - The username to log-in with.
+
+  password:
+    required: false
+    description:
+      - The password to log-in with.
+
+  id:
+    required: true
+    description:
+      - Unique identifier for the app consisting of a series of names separated by slashes.
+
+  command:
+    aliases: [ cmd ]
+    required: false
+    description:
+      - The command that is executed.
+
+  arguments:
+    aliases: [ args ]
+    required: false
+    description:
+      - An array of strings that represents an alternative mode of specifying the command to run.
+
+  cpus:
+    required: false
+    description:
+     - The number of CPU`s this application needs per instance. This number does not have to be integer, but can be a fraction.
+
+  memory:
+    aliases: [ mem ]
+    required: false
+    description:
+     - The amount of memory in MB that is needed for the application per instance.
+
+  disk:
+    required: false
+    description:
+     - The amount of disk in MB that is reserved for the application per instance.
+
+  ports:
+    required: false
+    description:
+     - An array of required port resources on the host.
+
+  requirePorts:
+    required: false
+    description:
+     - If true, the ports you have specified are used as host ports.
+
+  instances:
+    required: false
+    description:
+     - The number of instances of this application to start.
+
+  executor:
+    required: false
+    description:
+     - The executor to use to launch this application.
+
+  user:
+    required: false
+    description:
+      - The user to use to launch this application.
+
+  container:
+    required: false
+    description:
+     - Additional data passed to the containerizer on application launch. This is a free-form data structure that can contain arbitrary data.
+
+  docker_image:
+    required: false
+    description:
+     - Name of the Docker image. Ignored if container is defined.
+
+  docker_forcePullImage:
+    required: false
+    description:
+     - Force Docker to pull the image before launching each task. Ignored if container is defined.
+
+  docker_privileged:
+    required: false
+    description:
+     - Allows users to run containers in privileged mode. Ignored if container is defined.
+
+  docker_network:
+    required: false
+    description:
+     - Type of networking for the Docker container. Ignored if container is defined.
+
+  docker_portMappings:
+    required: false
+    description:
+     - Port mappings for the Docker container. Ignored if container is defined.
+
+  docker_parameters:
+    required: false
+    description:
+     - Arbitrary parameters for the Docker container. Ignored if container is defined.
+
+  container_volumes:
+    required: false
+    description:
+     - Array of volumes for the container. Ignored if container is defined.
+
+  env:
+    required: false
```

Comment:
if required: false is defined you must define a default e.g. default: null also see http://docs.ansible.com/ansible/developing_modules.html#module-checklist